### ISSUE TITLE
Critical vulnerabilities in node

### ISSUE THREAD

**Issue by digiserg** [axonops/axonops-workbench-cassandra#47]
```
up to date, audited 622 packages in 2s

118 packages are looking for funding
  run `npm fund` for details

4 vulnerabilities (1 low, 3 high)
```

**Comment by mhmdkrmabd** [comment]
Working on that

**Comment by mhmdkrmabd** [comment]
@digiserg all vulnerabilities have been addressed:
- High in `electron-builder`:
	- Depends on vulnerable versions of app-builder-lib and dmg-builder.
	- Fix: Updated the package to the latest versions.
- High in `app-builder-lib`:
	- Electron-builder's NSIS installer can execute arbitrary code on the target machine (Windows only).
	- Fix: Updated to electron-builder@24.13.3.
- High in `dmg-builder`:
	- Depends on vulnerable versions of `app-builder-lib`.
	- Fix: By fixing `app-builder-lib`, this also has been fixed indirectly.
- Low in `markdown`:
	- Regular Expression Denial of Service (ReDoS).
	- Fix: Removed the package and used a new one `marked`.

```
mhmd@mhmd-abd:axonops-workbench-cassandra$ npm i
> axonops-workbench-cassandra@0.1.0-internal-20240719 postinstall
> electron-builder install-app-deps

  • electron-builder  version=24.13.3
  • loaded configuration  file=package.json ("build" field)
  • rebuilding native dependencies  dependencies=better-sqlite3@11.1.2, cpu-features@0.0.10, fast-string-search@1.4.4, keytar@7.9.0, node-pty@1.0.0 platform=linux arch=x64
  • install prebuilt binary  name=keytar version=7.9.0 platform=linux arch=x64 napi=
  • install prebuilt binary  name=better-sqlite3 version=11.1.2 platform=linux arch=x64 napi=

up to date, audited 664 packages in 5s

119 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by digiserg** [comment]
Thanks.

Let me check electron-builder before we merge. There was a bug in some versions that broke the apple package signing.
